spec/mailers/abuse_mailer_spec.rb

```diff
@@ -87,11 +87,44 @@
    it "body text contains the comment" do
      expect(mail).to have_body_text(/#{report.comment}/)
    end
-
+   
    it "has the url of the page with abuse" do
      expect(mail.html_part).to have_body_text(report.url)
    end
   end
 
+  
+  context "abuse_reports contains creator and title if reporting a work" do
+   let(:work) {create(:work)}
+   let(:report) {create(:abuse_report, url: work_url(work.url)}
```

Comment:
What Hound is trying to say is that you have a missing bracket here - your `create()` call is missing its closing bracket. That line should be:
```
  let(:report) {create(:abuse_report, url: work_url(work.url))}
```